tapestry.php

```diff
@@ -316,3 +316,85 @@ function tapestry_tool_log_event()
 
     wp_die();
 }
+
+
+// Kaltura
+add_action('upload_videos_to_kalture', 'upload_videos_to_kalture');
+function upload_videos_to_kaltura()
+{
+    if (LOAD_KALTURA) {
+        $upload_folder = getcwd()."/wp-content/uploads";
```

Comment:
It would be more reliable to use the Wordpress [wp_upload_dir](https://developer.wordpress.org/reference/functions/wp_upload_dir/) function to get the upload dir